.github/workflows/build.yaml

```diff
@@ -0,0 +1,82 @@
+---
+name: build
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*.*.*
+  workflow_dispatch:
```

Comment:
I don't think that these triggers are quite what we want. We're trying to implement something equivalent to one of our Jenkins commit pipelines, which build and test pull requests. This configuration will only run when code is merged to main or released. At a minimum, we want to build on push to any branch ([see Vizr example](https://github.com/OCTRI/REDCap-Vizr/blob/master/.github/workflows/test.yml#L2-L5)).